.github/workflows/run-precommit.yml

```diff
@@ -1,14 +1,17 @@
 name: Run pre-commit hooks
 on:
   workflow_dispatch:
-  pull_request:
+  pull_request_target:
```

Comment:
why do we need to change it for the pre-commit?

Reply:
Pre-commit can actually run code from the repo, so I thought it's safer
(you can add scripts to the pre-commit config, or install a bad package via the requirements)

Reply:
Actually we really don't
I thought about that it's running code, but it's not using secrets so no issues here, changing back